core-web/libs/sdk/experiments/src/lib/models.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Represents the configuration for the SDK experiment.
+ *
+ * @interface SdkExperimentConfig
+ * @property {string} 'api-key' - The API key for the experiment.
+ * @property {string} server - The server URL of the backend.
+ * @property {boolean} debug - Specifies whether to enable debug mode for the experiment.
+ */
+export interface SdkExperimentConfig {
+    'api-key': string;
+    server: string;
+    debug: boolean;
```

Comment:
The same keys were present in the `constants.ts` file, although they had a prefix. Should we create a constant for these keys?

Reply:
There was an interface I didn't see.